scripts/gulp/live-reload-server.js

```diff
@@ -33,43 +42,103 @@ function LiveReloadServer(port, config) {
-      </script>
-    </body>
-    </html>
-      `;
+      var url = urlParser.parse(req.url);
+      var content;
+
+      if (url.pathname === '/document/license') {
+        content = `
+          <html>
+          <head>
+            <meta charset="UTF-8">
+            <title>Hypothesis in-line frame document - License</title>
+          </head>
+          <body>
+            <pre style="margin: 20px;">${licenseText()}</pre>
+          </body>
+          </html>
+        `;
+      } else if (url.pathname === '/document/changelog') {
+        content = `
+          <html>
+          <head>
+            <meta charset="UTF-8">
+            <title>Hypothesis in-line frame document - Changelog</title>
+          </head>
+          <body>
+            <pre style="margin: 20px;">${changelogText()}</pre>
+          </body>
+          </html>
+        `;
+      } else {
+        var multiFrameContent = config.enableMultiFrameSupport ? `
+          <div style="margin: 10px 0 0 75px;">
+            <button id="add-test" style="padding: 0.6em; font-size: 0.75em">Toggle 2nd Frame</button>
+          </div>
+          <div style="margin: 10px 0 0 75px;">
+            <iframe id="iframe1" src="/document/license" style="width: 50%;height: 300px;"></iframe>
+          </div>
+          <div id="iframe2-container" style="margin: 10px 0 0 75px;">
+          </div>` : '';
+
+        content = `
+          <html>
+          <head>
+            <meta charset="UTF-8">
+            <title>Hypothesis Client Test</title>
+          </head>
+          <body>
+            <div data-hypothesis-trigger style="margin: 75px 0 0 75px;">
+              Number of annotations:
+              <span data-hypothesis-annotation-count>...</span>
+            </div>
+            ${multiFrameContent}
+            <pre style="margin: 20px 75px 75px 75px;">${readmeText()}</pre>
+            <script>
+            var appHost = document.location.hostname;
+      
+            window.hypothesisConfig = function () {
+              return {
+                liveReloadServer: 'ws://' + appHost + ':${port}',
+      
+                // Open the sidebar when the page loads
+                openSidebar: true,
+                
+                // Needed for multi frame support
+                enableMultiFrameSupport: ${config.enableMultiFrameSupport},
+                embedScriptUrl: '${config.clientUrl}'
+              };
+            };
+      
+            window.addEventListener('message', function (event) {
+              if (event.data.type && event.data.type === 'reloadrequest') {
+                window.location.reload();
+              }
+            });
+      
+            var embedScript = document.createElement('script');
+            embedScript.src = '${config.clientUrl}';
+            document.body.appendChild(embedScript);
+            
+            var iframeIsAdded = false;
+            document.querySelector('#add-test').addEventListener('click', function() {
```

Comment:
document.querySelector('#add-test') is null when not in iframe test mode - causing this line to throw